src/Codebase/Infrastructure/ToolingConfigurationReader/ToolingConfigurationReader.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace Codebase\Infrastructure\ToolingConfigurationReader;
+
+use Codebase\Application\Dto\ConfigurationResponseDto;
+use Codebase\Infrastructure\Exception\ProjectConfigurationFileInvalidSyntaxException;
+use Exception;
+use Symfony\Component\Yaml\Yaml;
+
+class ToolingConfigurationReader implements ToolingConfigurationReaderInterface
+{
+    /**
+     * @var string
+     */
+    protected const UPGRADER_KEY = 'upgrader';
+
+    /**
+     * @var string
+     */
+    protected const PREFIXES_KEY = 'prefixes';
+
+    /**
+     * @var string
+     */
+    protected const DEFAULT_PREFIX = 'Pyz';
+
+    /**
+     * @param string $configurationFilePath
+     *
+     * @throws \Codebase\Infrastructure\Exception\ProjectConfigurationFileInvalidSyntaxException
+     *
+     * @return \Codebase\Application\Dto\ConfigurationResponseDto
+     */
+    public function readConfiguration(string $configurationFilePath): ConfigurationResponseDto
+    {
+        try {
+            $projectPrefixes = $this->parseProjectPrefixes($configurationFilePath);
```

Comment:
Why we parse just project prefixes, what about other configuration?

Reply:
we don't need other configuration inside tooling.yaml https://spryker.atlassian.net/browse/PPLUS-876